src/main/resources/messages.properties

```
@@ -647,9 +648,9 @@ activities-self-employment.title=Activities Self Employment
 activities-self-employment.header=Is your job here considered freelance, contracting, or self-employment?
 activities-self-employment.subtext=If yes, you can submit work-related expenses later in this application that can help with eligibility.
 #delete-jobs
-delete-job.title=Delete Jobs
-delete-job.header=Are you sure you want to delete {0} from your jobs?
-delete-job.yes=Yes, delete
+delete-job.title=Remove Jobs
+delete-job.header=Are you sure you want to remove {0} from your jobs?
+delete-job.yes=Yes,remove
```

A space is needed here:

```suggestion
delete-job.yes=Yes, remove
```

---

I have added the space and pushed the updated code.